docs/guides/slurm-hpc-ux.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: SPANK plugin user guide
+description: User guide for the quantum resource management SPANK plugin
+---
+{/* cspell:ignore QRMI, SBATCH, srun, Pasqal, slurmd, Doxyfile, Gres */}
+
+# SPANK plugin user guide
+
+Slurm QPU resource definitions determine what physical resources can be used by Slurm jobs in high-performance compute (HPC) environments. User source code should be agnostic to specific backend instances and even backend types as far as possible. This keeps source code portable while the QPU selection criteria are part of the resource definition (which is considered configuration as opposed to source code).
```

Comment:
```suggestion
Slurm QPU resource definitions determine what physical resources can be used by Slurm jobs in high-performance compute (HPC) environments. User source code should be agnostic to specific backend instances, and even backend types whenever possible. This keeps source code portable while the QPU selection criteria are part of the resource definition (which is considered configuration rather than source code).
```